linkatos.py

```diff
@@ -36,20 +43,21 @@ def keep_wanted_urls(expecting_confirmation, url):
         parsed_message,
         expecting_confirmation)
 
-    # Check if there is an answer
+    # check if there is an answer
     (expecting_confirmation, is_yes) = confirmation.process_confirmation_if_yn(
                                             parsed_message,
                                             expecting_confirmation)
 
     # printer.notify_confirmation(expecting_confirmation, is_yes)
 
     # Store url
-    # store_url(expecting_confirmation, url, is_yes)
+    is_yes = firebase.store_url(is_yes, url, FB_API_KEY, FB_USER, FB_PASS)
```

Comment:
To only store the url when the answer is 'yes'.
I wasn't sure how to keep that information overtime. Actually is_yes must be False at the start of every loop so the function should not be returning anything (?)